frontend/src/stores/levels/actions.ts

```diff
@@ -0,0 +1,17 @@
+import { levels } from './state'
+import { get } from '@/services/http'
+
+export async function fetchLevelsByRoleId(roleId: string) {
+  const data: any[] | string = await get(`/levels?role_id=${roleId}`)
```

Comment:
Never use any type. You are retrieving levels of a given role, right? So, what`s the type of your return?